packages/nouns-webapp/src/components/ExploreGrid/ExploreNounDetail/index.tsx

```diff
@@ -0,0 +1,366 @@
+import React, { ReactNode, useEffect, useState } from 'react';
+import { useNounSeed } from '../../../wrappers/nounToken';
+import { BigNumber } from 'ethers';
+import { StandalonePart } from '../../StandalonePart';
+import classes from './ExploreNounDetail.module.css';
+import { ImageData } from '@nouns/assets';
+import { Trans } from '@lingui/macro';
+import { AnimatePresence, motion } from 'framer-motion/dist/framer-motion';
+import { XIcon } from '@heroicons/react/solid';
+import NounInfoRowBirthday from '../../NounInfoRowBirthday';
+import loadingNoun from '../../../assets/loading-skull-noun.gif';
+import Placeholder from 'react-bootstrap/Placeholder';
+import Image from 'react-bootstrap/Image';
+import cx from 'classnames';
+import { useSwipeable } from 'react-swipeable';
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Already run in `index.ts`

Reply:
resolved in https://github.com/nounsDAO/nouns-monorepo/pull/595/commits/26745469ba53750e9b5cdf25f0d94368ea6f02ff